.teamcity/Database/buildTypes/DeployDatabase.kt

```diff
@@ -0,0 +1,48 @@
+// SPDX-License-Identifier: Apache-2.0
+// Licensed to the Ed-Fi Alliance under one or more agreements.
+// The Ed-Fi Alliance licenses this file to you under the Apache License, Version 2.0.
+// See the LICENSE and NOTICES files in the project root for more information.
+
+package database.buildTypes
+
+import jetbrains.buildServer.configs.kotlin.v2019_2.*
+import jetbrains.buildServer.configs.kotlin.v2019_2.buildFeatures.swabra
+import jetbrains.buildServer.configs.kotlin.v2019_2.buildFeatures.nuGetFeedCredentials
+import jetbrains.buildServer.configs.kotlin.v2019_2.buildSteps.nuGetPublish
+import jetbrains.buildServer.configs.kotlin.v2019_2.buildSteps.powerShell
+
+
+object DeployDatabaseBuild : BuildType ({
+    name = "Deploy"
+
+    features {
+        // Default setting is to clean before next build
+        swabra {
+        }
+
+        nuGetFeedCredentials {
+            feedUrl = "%azureArtifacts.feed.nuget%"
+            username = "%azureArtifacts.edFiBuildAgent.userName%"
+            password = "%azureArtifacts.edFiBuildAgent.accessToken%"
+        }
+    }
+
+    dependencies {
+        artifacts(PullRequestDatabaseBuild) {
```

Comment:
That's going to be a lot of deployments, if they are triggered off of the pull requests rather than the `main` branch builds. Some of deployed packages will contain code that is not yet accepted in the PR.

Reply:
In which build would I trigger the Nuget publish of the artifacts?